docs/docs/reference/attestations.md

```diff
@@ -0,0 +1,193 @@
+---
+sidebar_position: 1
+title: Contract-less and auto-discovery of materials
+---
+A Workflow Contract specifies the necessary content that a workflow must include in its attestation. For instance, it might mandate 
+the inclusion of the URI@digest of the generated container image, the container root filesystem used during the build, and the 
+Software Bill of Materials for that container image. These pieces of evidence must be associated with a specific material type. 
+Operators define what must be included in the contracts, and developers need to understand and comply with these requirements.
+
+This has been the case up until now, with the introduction of contract-less materials and auto-discovery, we ease the job of operators 
```

Comment:
Is there any reason for having multiple lines per paragraphs? It looks weird to me

Reply:
I believe it's just for clarity. I often do this. Note that final rendered result won't have any line feed.

Reply:
No really, will change it.

Reply:
mm, out of curiosity, how does this help with clarity? Wouldn't be better to have line-wrap in your editor instead? I find it a little bit unsettling to have break lines and then expect the parser to ignore them.

Reply:
Well, it seems that's an old discussion probably not worth having :) https://code.dblock.org/2021/06/07/to-wrap-or-not-to-wrap-in-markdown.html#:~:text=Here's%20a%20logical%20argument%20for,23%20characters%20for%20illustration%20purposes.